src/main/java/net/dv8tion/jda/internal/interactions/command/AutoCompleteCommandInteractionImpl.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.internal.interactions.command;
+
+import net.dv8tion.jda.api.interactions.AutoCompleteQuery;
+import net.dv8tion.jda.api.interactions.commands.*;
+import net.dv8tion.jda.api.requests.restaction.interactions.AutoCompleteCallbackAction;
+import net.dv8tion.jda.api.utils.data.DataArray;
+import net.dv8tion.jda.api.utils.data.DataObject;
+import net.dv8tion.jda.internal.JDAImpl;
+import net.dv8tion.jda.internal.interactions.InteractionImpl;
+import net.dv8tion.jda.internal.requests.restaction.interactions.AutoCompleteCallbackActionImpl;
+
+import javax.annotation.Nonnull;
+import java.util.Collection;
+
+public class AutoCompleteCommandInteractionImpl extends InteractionImpl implements CommandInteractionPayloadMixin, CommandAutoCompleteInteraction
```

Comment:
This file/class is named incorrectly. It should be `CommandAutoCompleteInteractionImpl`

Reply:
I'd also be okay changing all the other variants of this to align with _this_ file.
 - `AutoCompleteCommandInteraction`
 - `AutoCompleteCommandInteractionEvent`
 - `onAutoCompleteCommandInteraction`
 
 🤷

Reply:
I think the `CommandAutoComplete` makes more sense, doing `AutoCompleteCommand` sounds like its a command of type auto-complete, like we have SlashCommandInteraction.

Reply:
Yeah, that makes sense. So lets just realign this one misaligned file then.